app/src/main/java/com/example/concordia_campus_guide/Helper/IndoorPathHeuristic.java

```diff
@@ -82,7 +84,7 @@ protected WalkingPoint getClosestPointToCurrentPointFromList(final WalkingPoint
      * @param secondCoordinate
      * @return
      */
-    protected double getEuclideanDistance(final WalkingPoint firstCoordinate, final WalkingPoint secondCoordinate) {
+    public double getEuclideanDistance(final WalkingPoint firstCoordinate, final WalkingPoint secondCoordinate) {
```

Comment:
We need these changes in visibility for tests?

Reply:
yes because we can't access the protected or private methods.